full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -53,6 +54,58 @@ export namespace AzuriteTest {
       return blobClient.exists();
     };
 
+    export const subtractFromCurrentWriteLockExpiryTime = async (container: CloudSqlite.CloudContainer, blockName: string, numOfMin: number) => {
+      const azClient = createAzClient(container.containerId);
+      const blobClient = azClient.getBlockBlobClient(blockName);
+      const tempFilePath = join(KnownLocations.tmpdir, blockName);
+      // download bcv_kv.bcv to local machine
+      await blobClient.downloadToFile(tempFilePath);
+      const bcvDb = new SQLiteDb();
+      bcvDb.openDb(tempFilePath, {openMode: OpenMode.ReadWrite, rawSQLite: true});
+      // read writeLock.expires and modify it
+      bcvDb.withSqliteStatement("SELECT v FROM kv WHERE k = 'writeLock'", (stmt) => {
+        if (stmt.nextRow()) {
+          const writeLockData = JSON.parse(stmt.getValueString(0));
+          const expiresTime = new Date(writeLockData.expires);
+          expiresTime.setMinutes(expiresTime.getMinutes() - numOfMin);
+          writeLockData.expires = expiresTime.toISOString();
+          bcvDb.withSqliteStatement("UPDATE kv SET v = ? WHERE k = 'writeLock'", (stmt1) => {
+            stmt1.bindString(1, JSON.stringify(writeLockData));
+            stmt1.step();
+          });
+          bcvDb.saveChanges();
+        }
+      });
+      await blobClient.uploadFile(tempFilePath);
+      // clean up
+      bcvDb.closeDb();
+      fs.unlinkSync(tempFilePath);
+    };
+
+    export const isWriteLockValidForAtLeast = async (container: CloudSqlite.CloudContainer, blockName: string, currentTime: Date, writeLockTimeRemainsMs: number): Promise<boolean> => {
```

Comment:
same wtih this blockName, bcv_kv.bcv is implied to be the blockName since we're dealing with the writelock

Reply:
Resolved in [commit b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8](https://github.com/iTwin/itwinjs-core/pull/7000/commits/b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8)